src/azure-cli/azure/cli/command_modules/eventgrid/tests/latest/test_eventgrid_commands.py

```diff
@@ -566,6 +572,42 @@ def test_create_system_topic(self, resource_group):
 
         self.cmd('az eventgrid system-topic delete -n {system_topic_name} -g devexprg -y')
 
+    @ResourceGroupPreparer()
+    def test_system_topic_identity(self, resource_group):
+        scope = self.cmd('az group show -n {} -o json'.format(resource_group)).get_output_in_json()['id']
+        storage_system_topic_name_regional = self.create_random_name(prefix='cli', length=40)
+        policy_system_topic_name_global = 'policy-system-topic-name-global'
+
+        storage_account = '/subscriptions/5b4b650e-28b9-4790-b3ab-ddbd88d727c4/resourceGroups/devexprg/providers/Microsoft.Storage/storageAccounts/clistgaccount'
```

Comment:
Can you use `@StorageAccountPreparer()` instead of fixed storage account?

Reply:
We have some requirements on the storageaccount so we have one setup for all our tests specifically. i'm reusing that.